wordpress-mobile/gutenberg-mobile
# Overwrite Missing block UI Text

WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14880
WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/13011
gutenberg PR: https://github.com/WordPress/gutenberg/pull/25238

This PR implements the changes from https://github.com/WordPress/gutenberg/pull/25238 and overwrites the default Missing block alert text with text that references WordPress.com and Jetpack.

This also allows us to change these texts dynamically depending on the UBE-related capabilities sent from the native apps, after calling `updateCapabilities()`.

To test:
Please check https://github.com/wordpress-mobile/WordPress-iOS/pull/14880

PR submission checklist:
